client/src/extension.ts

```diff
@@ -40,24 +40,24 @@ export function activate(context: vscode.ExtensionContext) {
         // Name of the launcher class which contains the main.
         const main: string = 'org.drools.lsp.server.DroolsLspLauncher';
```

Comment:
Why do we do this: `java -cp drools-lsp-server-jar-with-dependencies.jar org.drools.lsp.server.DroolsLspLauncher`
instead of simply this: `java -jar drools-lsp-server-jar-with-dependencies.jar`?

Our `META-INF/MANIFEST.MF` file specifies `Main-Class: org.drools.lsp.server.Main`, so we don't have to specify the main class in the command, and just use the `-jar` syntax.

My follow-up question is, why do we have both `org.drools.lsp.server.DroolsLspLauncher` and `org.drools.lsp.server.Main`? They seem very similar...

Reply:
> Why do we do this: `java -cp drools-lsp-server-jar-with-dependencies.jar org.drools.lsp.server.DroolsLspLauncher` instead of simply this: `java -jar drools-lsp-server-jar-with-dependencies.jar`?

Thanks, @errantepiphany . I changed it to use `-jar`.

> My follow-up question is, why do we have both `org.drools.lsp.server.DroolsLspLauncher` and `org.drools.lsp.server.Main`? They seem very similar...

Thanks for pointing out. Yea, it looks like a duplicate and `DroolsLspLauncher` is not used at all. Removed.